addons/website/static/src/scss/website.scss

```diff
@@ -1306,6 +1306,7 @@ span.list-inline-item.o_add_language:last-child {
         $-copyright-color: o-color('copyright');
         $-copyright-color: mix(rgba($-copyright-color, 1.0), $-footer-color, percentage(alpha($-copyright-color)));
 
+        z-index: 1;
```

Comment:
You have an example where this is useful?

Reply:
@qsm-odoo Actually the bug does not exist in V14.0, it only happens from V15.0.. I will remove it in this Pr and create a new PR in V15.0 for it

Reply:
Actually adding `position: relative` on the `o_footer_scrolltop_wrapper` fixes the issue above (with `#footer`) but not below (the `.col` inside `o_footer_copyright` is still upper the scroll button.